pkg/logging/logging.go

```diff
@@ -104,12 +104,16 @@ func setFireLevels(level logrus.Level) []logrus.Level {
 	}
 }
 
+// InitializeDefaultLogger returns a logrus Logger with a custom text formatter.
+func InitializeDefaultLogger() *logrus.Logger {
```

Comment:
Could you also make a call to this at https://github.com/cilium/cilium/blob/master/pkg/endpoint/log.go#L62 so it also gets the correct formatter?
The endpoints end up with a separate logger object when their config is set to debug. In hindsight, a better solution to that problem would be a global DefaultDebugLogger that would be here, but that isn't the point of your PR.

Reply:
Utilized this function at the line as you requested.